packages/server/graphql/types/AddPokerTemplatePayload.ts

```diff
@@ -15,6 +17,14 @@ const AddPokerTemplatePayload = new GraphQLObjectType<any, GQLContext>({
         if (!templateId) return null
         return dataLoader.get('meetingTemplates').load(templateId)
       }
+    },
+    user: {
```

Comment:
I'll refactor this to the SDL pattern in a follow-up PR. This PR already touches a lot of files, so I didn't want to bloat it further